test/tools/libtesteth/TestHelper.cpp

```diff
@@ -242,21 +243,31 @@ void replaceLLLinState(json_spirit::mObject& _o)
 	}
 }
 
-std::vector<boost::filesystem::path> getJsonFiles(boost::filesystem::path const& _dirPath, std::string const& _particularFile)
+std::vector<boost::filesystem::path> getFiles(boost::filesystem::path const& _dirPath, string const& _extentionMask, std::string const& _particularFile)
 {
 	vector<boost::filesystem::path> jsonFiles;
+	vector<string> extentions;
+	boost::split(extentions, _extentionMask, boost::is_any_of("|"));
 	if (!_particularFile.empty())
 	{
-		boost::filesystem::path file = _dirPath / (_particularFile + ".json");
-		if (boost::filesystem::exists(file))
-			jsonFiles.push_back(file);
+		for(auto const& ext: extentions)
```

Comment:
If `extensions` contains duplicates you will add the same file many times.

Reply:
true